docs/platform_concepts/inbox/inbox-settings/chats-configuration/tags.md

```diff
@@ -1,4 +1,33 @@
 ---
 title: Tags
 sidebar_label : Tags
----
\ No newline at end of file
+---
+
+
+Tags are labels that you assign to conversations after the end or during the chat sessions and one of the best ways to organize conversations and for quicker follow-ups. They are typically used to categorize and organize all customer communication and to escalate it to the right team for further actions fast and effectively.
+
+Tagging your chats helps you search or filter your chat menu and makes it easier to analyze the chat traffic.
+
+
+![](https://i.imgur.com/39vPpD6.png)
+
+
+#### Learn how to use tags
```

Comment:
Use level 3 header (### HEADER HERE) to display it in the right sidebar

Reply:
Done.